clusters/app.ci/vault/manifests.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vault
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+    route.openshift.io/termination: passthrough
+  name: vault
+  namespace: vault
+spec:
+  tls:
+    - secretName: vault-tls
+      hosts:
+        - vault.ci.openshift.org
+  rules:
+  - host: vault.ci.openshift.org
+    http:
+      paths:
+      - backend:
+          service:
+            name: vault
+            port:
+              number: 8200
+        pathType: ImplementationSpecific
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: vault
+  namespace: vault
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - name: 1.6.2
+    from:
+      kind: DockerImage
+      # Mirror of docker.io/library/vault:1.6.2 because the cluster hit the ratelimit for that
+      name: quay.io/alvaroaleman/vault:1.6.2
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: vault
+  name: vault
+  namespace: vault
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: vault
+  template:
+    metadata:
+      labels:
+        app: vault
+      annotations:
+        config.hcl: |
+          api_addr = "https://vault.ci.openshift.org"
+          listener "tcp" {
+            address       = "0.0.0.0:8200"
+            # TODO: Sidecar that sends sighub if they change so vault reloads, otherwise
+            # triage will have some fun in three months
+            tls_cert_file = "/var/run/serving-cert/tls.crt"
+            tls_key_file  = "/var/run/serving-cert/tls.key"
+          }
+
+          storage "gcs" {
+            bucket        = "vault-ci-openshift"
+            ha_enabled    = "true"
+          }
+          disable_mlock = "true"
+          ui = "true"
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - vault
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+      - image: vault:1.6.2
+        name: unsealer
+        command:
+        - /bin/sh
+        - -c
+        - |-
+          set -u
+          export VAULT_ADDR=https://127.0.0.1:8200
+          # Cert is only valid for https://vault.ci.openshift.org, not 127.0.0.1
+          export VAULT_SKIP_VERIFY=true
+          until vault status|grep Sealed|grep -q false; do
```

Comment:
Looks like we can just use the exit code:
> The exit code reflects the seal status:
> 0 - unsealed
> 1 - error
> 2 - sealed

https://www.vaultproject.io/docs/commands/status

Reply:
Updated